Felienne/hedy
# [UI IMPROVEMENT] Makes error line highlighting more readable

**Description**
In this PR we make the line error highlighting more readable. Due to the keywords as well as the line being red it was hard to debug when an error was made while there was also keyword highlighting in the same line. We fix this by changing the highlighting from a red background to a red top and bottom border, see the screenshot below for clarification. We also remove the Red Cross on the gutter as it was a bit redundant and overlayed the line counter.

![Schermafbeelding 2022-07-06 om 13 37 48](https://user-images.githubusercontent.com/38155424/177541931-6bfdff7c-2edf-4d24-b66c-1129f736ce93.png)

**Fixes**
This PR fixes #2955

**How to test**
Create a program with a mistake and notice the new line error highlighting. Verify that this works in all cases.
